stats.tex

```diff
@@ -0,0 +1,50 @@
+%% Stats
+The \verb|sympy.stats| module provides random variable types and methods for
+determination of statistical properties of expressions involving random
+variables. The variables can be either continuous or discontinuous, which are
+further divided into finite and infinite. They are associated
+with probability densities on corresponding domains and internally defined
+in terms of probability spaces. 
+Apart from the possibility of defining the random variables from user supplied
+density distribution, SymPy provides definitions of most common
+distributions, including \texttt{Uniform}, \texttt{Poisson}, \texttt{Normal},
+\texttt{Binomial}, \texttt{Bernoulli}, and many others.
+
+Properties of random expressions can be calculated using, e.g.,
+\texttt{E} and \texttt{variance} to calculate expectation and variance.
+Conditions on random variables can be defined with inequalities, equalities,
+and logical operators and their overall probabilities are obtained using
+\texttt{P}. The features can be illustrated on a model of two dice throws:
```

Comment:
probability( ) abbreviated P( ). I would like to change _P_ in future versions of SymPy to return an unevaluated probability expression.